README.md

```diff
@@ -245,23 +245,23 @@ Sets the request priority. Higher priority items will be scanned first.
 
 ```json
 {
-  "id": "c88057777737115851ad94b91461d09b2ce704484813557895ba0d6d827d4ed8",
+  "id": "tMd_Av-ZTFP",
   "request": {
-    "id": "580efb11ee9013a42d6037566c7159c9aeb610696be851dc9209c85e75e5a3e7",
+    "id": "Dg9hdRRdd2M",
     "kind": "JSONData",
     "resource": "{\"some\":{\"key\": \"-----BEGIN PRIVATE KEY-----c5602d28d0f21422dfc7b572b17e6b138c1b49fd7f477d4c5c961e0756f1ff70-----END PRIVATE KEY-----\"}}"
   },
   "logs": [],
   "results": [
     {
-      "id": "66456b43e1efac03f9448ece59a65b0e2bf304b55506507a8aa07727e3900522",
+      "id": "ZwaaqRjgNgk",
       "kind": "JSONData",
       "secret": "-----BEGIN PRIVATE KEY-----c5602d28d0f21422dfc7b572b17e6b138c1b49fd7f477d4c5c961e0756f1ff70-----END PRIVATE KEY-----",
       "match": "-----BEGIN PRIVATE KEY-----c5602d28d0f21422dfc7b572b17e6b138c1b49fd7f477d4c5c961e0756f1ff70-----END PRIVATE KEY-----",
       "entropy": 4.490795,
       "date": "",
       "rule": {
-        "id": "private-key",
+        "id": "3fk1rL-aRiw",
```

Comment:
I've always wondered, do these IDs have a semantic meaning? Are you matching them with IDs generated in another system?

What is the benefit of `3fk1rL-aRiw` over `private-key`?

Reply:
For the rule IDs, the reason we just have them as random strings instead of human readable tags is because:

- We'll probably add other backends and I don't want the IDs to collide, even of rules look for similar things.
- Naming is hard lol. If I change a rule's description because I realize it's not quite accurate and the id was based on what I thought a thing was called, I would be misnamed a bit. This happens sometimes when I'm trying to find the correct name for an API cred.
- We'll end up using the rule ID with other fields to help make sure we don't double send alerts to folks

The other IDs like request ID are used for mapping requests to responses in our alerting system and others are only currently used for tracing in our logs.